### ISSUE TITLE
[Question] Where to find the definitions of the terms shown in training?

### ISSUE THREAD

**Issue by lesshaste** [DLR-RM/stable-baselines3#739]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.

### Question

I was looking for documentation for definitions of the terms used in training, e.g.

```
-----------------------------------------
| time/                   |             |
|    fps                  | 2447        |
|    iterations           | 3           |
|    time_elapsed         | 30          |
|    total_timesteps      | 73728       |
| train/                  |             |
|    approx_kl            | 0.012046688 |
|    clip_fraction        | 0.124       |
|    clip_range           | 0.2         |
|    entropy_loss         | -2.28       |
|    explained_variance   | 0.58        |
|    learning_rate        | 0.0003      |
|    loss                 | 0.31        |
|    n_updates            | 20          |
|    policy_gradient_loss | -0.0174     |
|    value_loss           | 1.39        |
-----------------------------------------
```

I could not find anything. As an example for value_loss all I got was a link to the source code for PPO which says

```
# Value loss using the TD(gae_lambda) target
value_loss = F.mse_loss(rollout_data.returns, values_pred)
```

This gives a clue but doesn't tell me the whole story.  Is there any documentation for these terms I am missing?

**Comment by Miffyli** [comment]
You are right there is no documentation on these values as they change between algorithms. On one hand I can see how this is useful (helps people to understand what is happening), but on the other hand it will be more difficult to maintain (need to remember to include description for new variables if needed). Overall I think docs should have a description for these metrics, though :). A PR on this could possibly be accepted but I would wait for @araffin 's comment on the subject.

**Comment by duttashi** [comment]
@Miffyli  I agree and I feel its very important to have elaborate documentation related to the results generated by algorithms implemented in SB3. Absence of which may render fewer users of SB3 library. For instance, consider the Scikit-learn library for machine learning. One reason for its popularity is the extensive documentation pertaining to algorithms in code format. I've implemented PPO in SB3 and I've no idea how to interpret the results. Its like black magic and honestly does not go in good taste, if I were to use it in production!

**Comment by araffin** [comment]
> This gives a clue but doesn't tell me the whole story. Is there any documentation for these terms I am missing?

@lesshaste 
Regarding the value loss, I'm not sure what more we can add... (we already have link to resources to learn more about RL in our doc).
For the other values, I will try to a do a PR soon or in the meantime, write their meaning in #662 (or on request).

**Comment by araffin** [comment]
@lesshaste @duttashi I added short explanations in https://github.com/DLR-RM/stable-baselines3/pull/803, could you have a look?
